timesketch/lib/sigma_util.py

```diff
@@ -47,9 +45,13 @@ def get_sigma_config_file(config_file=None):
     else:
         config_file_path = current_app.config.get('SIGMA_CONFIG')
 
-    if not config_file:
-        raise ValueError(
-            'SIGMA_CONFIG not found in config file')
+    # this is mostly to be able to test cases via API where the test from
+    # API client
+    if not config_file_path:
+        config_file_path = './data/sigma_config.yaml'
```

Comment:
why do this?

you can also do 

```
current_app.config.get('SIGMA_CONFIG', './data/sigma_config.yaml')
```

or simply make sure that you pass in `./data/sigma_config.yaml` in when you do tests.... I would not have this in the production code where you are pointing to a test data set.

Reply:
yeah this is actually not about test data set but where the "default" location is stored, so the proposal with 

```current_app.config.get('SIGMA_CONFIG', './data/sigma_config.yaml')
``` is perfect.